site/docs/samples/notebook-read.md

```diff
@@ -61,85 +61,93 @@ Make a note of the service endpoint, bucket name, and access credentials. You wi
+
+    Store the asset ID in a `CATALOGED_ASSET` variable. For instance:
+    ```bash
+    CATALOGED_ASSET="openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""
+    ```
+
+=== "Registering Dataset via Connector"
+    We now explain how to register a dataset using the OpenMetadata connector. Begin by registering the credentials required for accessing the dataset as a kubernetes secret. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+
+    ```yaml
+    cat << EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: paysim-csv
+    type: Opaque
+    stringData:
+      access_key: "${ACCESS_KEY}"
+      secret_key: "${SECRET_KEY}"
+    EOF
+    ```
+
+    Next, register the data asset itself in the data catalog.
+    We use port-forwarding to send asset creation requests to the OpenMetadata connector.
+    ```bash
+    kubectl port-forward svc/openmetadata-connector -n fybrik-system 8081:8080 &
+    cat << EOF | curl -X POST localhost:8081/createAsset -d @-
+    {
+      "destinationCatalogID": "openmetadata",
+      "destinationAssetID": "paysim-csv",
+      "credentials": "/v1/kubernetes-secrets/paysim-csv?namespace=fybrik-notebook-sample",
+      "details": {
+        "dataFormat": "csv",
+        "connection": {
+          "name": "s3",
+          "s3": {
+            "endpoint": "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566",
+            "bucket": "demo",
+            "object_key": "PS_20174392719_1491204439457_log.csv"
+          }
         }
       },
-      {
-        "name": "newbalanceOrig",
+      "resourceMetadata": {
+        "name": "Synthetic Financial Datasets For Fraud Detection",
+        "geography": "theshire ",
         "tags": {
-          "PII": "true"
-        }
+          "finance": "true"
+        },
+        "columns": [
+          {
+            "name": "nameOrig",
+            "tags": {
+              "PII": "true"
+            }
+          },
+          {
+            "name": "oldbalanceOrg",
+            "tags": {
+              "PII": "true"
+            }
+          },
+          {
+            "name": "newbalanceOrig",
+            "tags": {
+              "PII": "true"
+            }
+          }
+        ]
       }
-    ]
-  }
-}
-EOF
-```
-
-The response from the OpenMetadata connector should look like this:
-```bash
-{"assetID":"openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""}
-```
-Store the asset ID in a `CATALOGED_ASSET` variable:
-```bash
-CATALOGED_ASSET="openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""
-```
+    }
+    EOF
+    ```
+
+    The response from the OpenMetadata connector should look like this:
+    ```bash
+    {"assetID":"openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""}
+    ```
```

Comment:
I added the following sentence:


If you look at the asset creation request above, you will notice that in the `resourceMetadata` field, we request that the asset should be tagged with the `finance` tag, and that three of its columns should be tagged with the `PII` tag. Those tags will be referenced below in the access policy rules. Tags are important because they are used to determine whether an application would be allowed to access a dataset, and if so, which transformations should be applied to it.